pontoon/base/views.py

```diff
@@ -1090,29 +1090,18 @@ def request_locale(request):
         log.error(str(e))
         return HttpResponse("error")
 
-    try:
-        project = Project.objects.get(slug=project)
-    except Entity.DoesNotExist as e:
-        log.error(str(e))
-        return HttpResponse("error")
-
-    try:
-        locale = Locale.objects.get(code__iexact=locale)
-    except Locale.DoesNotExist as e:
-        log.error(str(e))
-        return HttpResponse("error")
-
-    subject = '[Pontoon] Locale Request'
-    message = 'Add locale %s (%s) to project %s (%s).' % (
-        locale.name, locale.code, project.name, project.slug)
-    sender = request.user.email
+    project = get_object_or_404(Project, slug=project)
+    locale = get_object_or_404(Locale, code__iexact=locale)
 
     if settings.ADMINS[0][1] != '':
```

Comment:
Documentation says an empty tuple: https://docs.djangoproject.com/en/1.8/ref/settings/#admins

But we do override it in `base.py`. Huh. Do whatever you think is best, then. I figure being defensive about it is safer but it's not a big deal either way.